reference-implementation/lib/abstract-ops/queue-with-sizes.js

```diff
@@ -37,9 +39,23 @@ exports.PeekQueueValue = container => {
   return pair.value;
 };
 
+const closingStepsSymbol = Symbol('closing-steps');
```

Comment:
This symbol isn't exposed anywhere, so I suppose we can't really write tests for this? 😕

Reply:
It is difficult to write tests except when not running closing steps would have a functional effect (this is the case for some VideoFrame sources).
We could add some specific tests, but they would not be WPT tests.

Reply:
Let's hope the [explicit resource management proposal](https://github.com/tc39/proposal-explicit-resource-management) advances quickly, so we can integrate with something like `Symbol.dispose` instead. 😁

Reply:
It might be better to rename closing-steps to dispose-steps, I'll do this.